browser-test/bin/fmt

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# find absolute path to local browser-test directory
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && cd .. && pwd )"
+
+# find relative path to typescript files
+FILES="$(cd $DIR; find src -name '*.ts' -not -path '*node_modules*' -not -path '*node-modules*' | xargs)"
```

Comment:
Question (blocking): why does `-not -path '*node_modules*'` appear twice?

Reply:
Removed.